test.js

```diff
@@ -54,10 +55,10 @@ describe("bit-docs-html-codepen-link", function() {
 				browser.window.CREATE_CODE_PEN = function(data) {
 					createCallData.push(data);
 				};
-				var codePens = doc.querySelectorAll('.codepen');
-
-				Array.from(codePens).forEach(function(codePen) {
-					codePen.click();
+				var toolbars = doc.querySelectorAll('.toolbar');
+				Array.from(toolbars).forEach(function(toolbar) {
```

Comment:
Is `Array.from` necessary? I think `toolbars.forEach` would exist.

Reply:
No I just changed the existed code that used it, I'll update it